build.js

```diff
@@ -32,7 +31,27 @@ var path = require('path');
 var cheerio = require('cheerio');
 // var child_process = require('child_process');
 
-var useCompilers = String(process.argv[2]).toLowerCase() === "compilers";
+var useCompilers = false;
+var excludeCurrentBrowser = false;
+var environments;
+
+process.argv.slice(2).forEach(function(arg) {
+	var parts = String(arg).toLowerCase().split('=');
+
+	switch (parts[0]) {
+		case 'compilers':
+			useCompilers = true;
+			break;
+		case 'environments':
+			environments = require(parts[1]);
+			break;
+		case 'excludecurrent':
+			excludeCurrentBrowser = true;
+			break;
+	}
+});
```

Comment:
let's use an args parser instead of rolling our own - https://www.npmjs.com/package/@pkgjs/parseargs will eventually be built into node, so that might be a good choice

Reply:
Looking at the example for the parseargs package, what would be your preferred setup? 
- keep `compilers` as a positional?
- excludecurrent as a flag? If so, suggestions as to which one? `-e`?

So something like `-e --environments=./custom.json compilers`?

Reply:
That seems fine to me (altho maybe x for exclude)